css/css-tables-3/visibility-collapse-rowspan-002.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
```

Comment:
Can you please create a copy of `rowspan-002` & `rowspan-003` and create ones that are in `border-collapse: seperate` mode? Make sure this is reflected in the name so it's easy to differentiate the tests.